mayastor/tests/mount_fs.rs

```diff
@@ -169,9 +183,14 @@ fn mount_fs_2() {
         let (s, r) = unbounded::<String>();
 
         Mthread::spawn_unaffinitized(move || {
-            s.send(common::fio_run_verify(&device))
+            s.send(match common::fio_run_verify(&device) {
+                Ok(_) => "".to_owned(),
+                Err(err) => err,
+            })
         });
-        reactor_poll!(r);
+        let result;
```

Comment:
I see we use now this pattern a lot, maybe we can add it as a function in the test library?

Reply:
Yeah, but this whole reactor_poll!() stuff isn't really needed anymore, to begin with. I think @jkryl is focussing on getting the tests to pass reliably on the new environment.

Reply:
I have contributed a new macro which is a copy of reactor_poll!() and does also the assert - so all combined together. So that will be a small convenience to eliminate repeating code. I think that more fundamental changes can be done later.

Reply:
:+1: Let's get the tests running reliably first then we'll be able to refactor with more confidence :)